profile/nodejs.sh

```diff
@@ -24,10 +29,13 @@ detect_memory() {
 
 export PATH="$HOME/.heroku/node/bin:$HOME/bin:$HOME/node_modules/.bin:$PATH"
 export NODE_HOME="$HOME/.heroku/node"
-export NODE_ENV=${NODE_ENV:-production}
 
 calculate_concurrency
 
 export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
 export WEB_MEMORY=$WEB_MEMORY
 export WEB_CONCURRENCY=$WEB_CONCURRENCY
+
+if [ "$LOG_CONCURRENCY" = "true" ]; then
```

Comment:
I have not been able to reproduce the issue; none of these logs ("Creating Asset environment, etc") are strings that exist either in this buildpack or in the master branch of the linked buildpack.

It sounds like the other buildpack might be inadvertently trying to source nodejs.sh with `set -o nounset`.